docs/build-module-cr.md

```diff
@@ -0,0 +1,34 @@
+| [Home](../README.md) |
+|----------------------|
+
+## CR for Building a new Module
+
+This task entails following sub-tasks:
+
+1. Application administrator creates a new CR for building a new module. For creating a new CR refer to the [New Change Request from Production Environment](#new-change-request-from-production-environment).
```

Comment:
Application administrator creates a new CR in Production Environment for building a new module.